src/Conversion/KrnlToAffine/KrnlToAffine.cpp

```diff
@@ -763,6 +763,7 @@ class KrnlMatmulLowering : public OpRewritePattern<KrnlMatMulOp> {
       simdize = false;
       LLVM_DEBUG(llvm::dbgs() << "Matmul: No simd due to vl not a literal\n");
     }
+#if 0
```

Comment:
Given this is not needed I think it would be cleaner to remove the code all together (this is usually what is done in LLVM).

Reply:
I will remove it after a round of successful test.